eval/builtin_fn_misc.go

```diff
@@ -60,6 +63,19 @@ func constantly(args ...interface{}) Callable {
 	)
 }
 
+func getenv(fm *Frame, key string) error {
+	value, exists := os.LookupEnv(key)
+	if !exists {
+		return errors.New("Non-existent environment variable")
+	}
+	fm.ports[1].Chan <- value
+	return nil
+}
+
+func setenv(fm *Frame, key, value string) error {
```

Comment:
Frame is also optional here. In fact, this entire function can be removed, just use os.Setenv.

Reply:
Done